src/Stratis.Bitcoin.Features.BlockStore/AddressIndexing/AddressIndexer.cs

```diff
@@ -457,6 +460,10 @@ private bool ProcessBlock(Block block, ChainedHeader header)
         }
 
         /// <summary>Adds a new balance change entry to to the <see cref="addressIndexRepository"/>.</summary>
+        /// <param name="height">The height of the block this being processed.</param>
+        /// <param name="address">The address receiving the funds.</param>
+        /// <param name="amount">The amount being received.</param>
+        /// <param name="deposited"><c>true</c> if this is an output being spent, <c>false</c> otherwise.</param>
```

Comment:
I think this might be backwards, a deposit is an incoming transaction for an address?

Reply:
@zeptin it seems we pass false here when call process from an TxInput and true from an TxOutput?

Reply:
Yes, but bear in mind that when we see a TxIn for a given scriptPubkey (address), that means that the prevOut in question is getting spent. So it is a withdrawal 'from' the address and therefore not a deposit. Conversely with the TxOut.

Reply:
OK fixed :) please check 👍